larray/tests/test_array.py

```diff
@@ -855,18 +855,6 @@ def test_setitem_larray(self):
         # same raw as previous test
         assert_array_equal(la, raw)
 
-        # c) value has an extra length-1 axis
```

Comment:
I wonder if keeping this behaviour would be worth it. Adapating the test would be trivial (exclude length 1 axes from extra_axes). A setitem with extra length-1 axes is non-ambiguous after all. I don't see a use case for it though, so it's your call.